### ISSUE TITLE
Demos not working

### ISSUE THREAD

**Issue by tobya** [jqueryfiletree/jqueryfiletree#62]
The demos on your demo page are not working.  they are showing an 'Unable to get file tree inforamtion' error.

The console shows a cross origin request being blocked to davidroge.rs/jqft/jqueryFileTree.php

**Comment by daverogers** [comment]
Yeah, someone was hitting my web server was getting hit like crazy. I need to isolate the demo in another server.

**Comment by fjeddy** [comment]
If that would be done in 2017 then that would be great :) Skipping this due to the lack of a demo.

**Comment by CodingSamurai** [comment]
I agree this needs to get fixed sooner rather than later if you want people to use this project.  We needed to make a decision today and without demos we'll just go with another library.

**Comment by dwvisser** [comment]
On a related note, the manual test instructions in the README don't really work to set up a local demo page, without some sort of server processing the `connectors/jqueryFileTree.php` server-side code. The README could do with some instructions on this, or switch to some embedded JavaScript function serving up some dummy file tree data.

**Comment by dwvisser** [comment]
Actually, I wanted to test my pull request #71, and realized that since I have Docker on my system, it's pretty easy to get an Apache HTTPD/PHP stack up and running in a container. Here the steps that worked for me, after building the manual test folder as described in the README:

    $ cd tests/manual
    $ docker run -d -p 80:80 --name my-apache-php-app -v "$PWD":/var/www/html php:7.0-apache 

Then, I just browsed to `http://localhost/index.html`. See https://hub.docker.com/_/php/ for more details on the Docker image used.
